ringpop-admin.js

```diff
@@ -38,6 +38,8 @@ function main() {
         .command('status', 'Status of members in ring')
         .command('partitions', 'Show partition information of a ring')
         .command('top', 'General membership information')
+        .command('reap', 'Remove the faulty nodes from this nodes membership')
+        .command('heal', 'Start a partition heal coordinated by the target node')
```

Comment:
There should not be a case when the heal doesn't work on its own, unless we made a mistake somewhere, or the moons align just right. The primary reason for me to add a heal command to ringpop-admin is to give complete control to an operator of a ringpop application.

In the best case they would never have to call this by hand because ringpop recovers on its own. But having this command just in case it doesn't an operator has the option to heal without restarting a service.

When an operator needs to do this operation it should be as easy as possible, without the need to lookup the right endpoint that needs to be called via tcurl. Remember that when this needs to be executed it is on high presure, a pagerduty alert came in that ringpop is partitioned. It should be idiomatic that the operator can use `ringpop-admin paritions` first to see what happened, and then call `ringpop-admin heal` as a first attempt to fix the partition.

I completely agree with @dansimau that `ringpop-admin` should hide the complexity from the user to invoke it multiple times. I thought that this endpoint would heal the partition, but it just tries it once I understand from @CorgiMan 's comment. Even a better argument to have this in `ringpop-admin` over running tcurl by hand.

I will work on calling the endpoint multiple times in a row until there is no partition to heal anymore.